docs: update preprocessors description (#1113)
Redocly/redocly-cli: docs/resources/custom-plugins.md

Use when you need to transform your API definition prior to validation.
Preprocessors are brittle and error prone because validation occurs **after** preprocessing.

> Please mind that preprocessors can impact performance by adding an extra resolution step.

Reviewer: ```suggestion
Please be aware that preprocessors can impact performance because they add an extra resolution step.
```

> Please be aware that preprocessors can impact performance because they add an extra resolution step.

Reviewer 2: ```suggestion
Preprocessors can impact performance because they add an extra resolution step.
```
We try to avoid words like "Please" in the docs.
